tests/src/test/java/org/wso2/micro/gateway/tests/common/CLIExecutor.java

```diff
@@ -64,6 +64,7 @@ public void generate(String label, String project) throws MicroGWTestException {
         runImportCmd(mgwCommand, project, importCmdArray);
         copyCustomizedPolicyFileFromResources(project);
         runBuildCmd(mgwCommand, project);
+        runVersionCmd(mgwCommand);
```

Comment:
What is the reason for adding this here? To know the version running during the tests? If that is the case shall we move this to line 64, before `runImportCmd`? also shouldn't this be added to `generateFromDefinition` too?

Reply:
yes. this is to know the version running during tests. Yeah it should be moved before `runImportCmd`. I will fix that.   I thought in the both cases it does the same thing, since I added this only once